src/components/HelFormFields/HelTextField.js

```diff
@@ -212,18 +213,17 @@ class HelTextField extends React.Component {
         if(this.props.type) {
             type = this.props.type
         } else {
-            type = this.props.multiLine ? 'textarea' : 'text'
+            type = this.props.multiLine ? 'textarea' : 'input'
```

Comment:
by default, type will be `input`, equivalent of `text`
`componentClass` is deprecated, will throw error